### DISCUSSION TITLE
Customize color for each level while enable 'Rainbowify tag name in XML/HTML'

### DISCUSSION THREAD

**Discussion by nanlan2017** [izhangzhihao/intellij-rainbow-brackets#2645] [Q&A] [answered]
I primarily use this plugin to create a rainbow effect on the tag hierarchy in HTML, specifically the arrow brackets. 
When I customize the colors of the arrow brackets at different levels (I want each level to have distinct colors rather than a smooth transition), enabling 'Rainbowify tag name in XML/HTML' option uses default colors that cannot be customized. 
This results in the '<' symbol of the '<tag>' being one color (customized), while the 'tag' itself is a different color (default). 
Is there a way to make these two consistent?

example as below:
![rainbow](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/25033914/30d65a8c-396a-4736-bf81-aa33716208ec)

**Comment by izhangzhihao** [comment, marked as answer]
To make the two consistent, please install this build snapshot

To config the XML tag colors:
<img width="1143" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/430d06b6-932c-4960-989a-a25dd0498ce2">

**Reply by nanlan2017** [reply]
It solved my problem. Thank you for your quick support, amazing!

**Reply by izhangzhihao** [reply]
Hi, your issue is already resolved, could you please update your review? https://plugins.jetbrains.com/plugin/10080-rainbow-brackets/reviews#review=85774

**Reply by nanlan2017** [reply]
Yes, I can't edit my review, so I just remove it. Thanks again!

**Reply by izhangzhihao** [reply]
Please leave 5 star review if you think this plugin helps, thanks😆

**Reply by nanlan2017** [reply]
Sure!
